apps/teamsjs-cc-app/tabs/src/helpers/writetoexcel.ts

```diff
@@ -0,0 +1,26 @@
+import { utils, write, writeFile } from "xlsx";
+
+import { ICapabilityStatus } from "../components/Tab";
+
+/**
+ * Creates a .csv file
+ * @param defaultRowList, type of ICapabilityStatus[]
```

Comment:
These descriptions of the parameters don't appear to provide enough information for a consumer to know how to use them. They literally just restate the parameter name and type, which someone could also get by just looking at the function signature.